hvplot/util.py

```diff
@@ -473,8 +473,9 @@ def process_intake(data, use_dask):
 def is_geodataframe(data):
     if 'spatialpandas' in sys.modules:
         import spatialpandas as spd
+        from spatialpandas.dask import DaskGeoDataFrame
```

Comment:
Does spatialpandas always include dask as dep? If not, this might crash.

Reply:
> Maybe you can just use `spd.dask.DaskGeoDataFrame` instead of another import for consistency, but no strong opinion.

Yeah, I think that is better.